internal/models/linking_test.go

```diff
@@ -82,18 +82,6 @@ func (ts *AccountLinkingTestSuite) TestCreateAccountDecisionWithAccounts() {
 	require.NoError(ts.T(), err)
 
 	require.Equal(ts.T(), decision.Decision, CreateAccount)
-
-	// when looking for an email that doesn't exist at all
-	decision, err = DetermineAccountLinking(ts.db, "sso:f06f9e3d-ff92-4c47-a179-7acf1fda6387", "abcdefgh", []string{"other@samltest.id"})
-	require.NoError(ts.T(), err)
-
-	require.Equal(ts.T(), decision.Decision, CreateAccount)
-
-	// when looking for an email that doesn't exist in the SSO linking domain
-	decision, err = DetermineAccountLinking(ts.db, "sso:f06f9e3d-ff92-4c47-a179-7acf1fda6387", "abcdefgh", []string{"text@example.com"})
-	require.NoError(ts.T(), err)
-
-	require.Equal(ts.T(), decision.Decision, CreateAccount)
```

Comment:
Just wandering why these were deleted? Seems unrelated to the changes.

Reply:
i was also testing the account linking behaviour and noticed that these tests were repeated so i decided to remove them